src/targets/github.ts

```diff
@@ -111,7 +111,8 @@ export class GitHubTarget extends BaseTarget {
   public async createDraftRelease(
     version: string,
     revision: string,
-    changes?: Changeset
+    changes?: Changeset,
+    isLatest = true
```

Comment:
```suggestion
    setAsLatest = true
```

Also, not to derail your patch but boolean function arguments are frowned upon as they make it very hard to see what's going on from the callsite: `createDraftRelease('1.2.3', 'aaaef32', undefined, false)` would be quite hard to make sense of.

You may want to consider converting all of them to an arguments object or just add a final `options` object with `setAsLatest` as its only known key for now?

Reply:
happy to do that, I tried to do minimal changes in the PR, but I agree - I'll convert this to an object!